bosch_thermostat_client/gateway/base.py

```diff
@@ -90,11 +90,11 @@ async def initialize(self):
             )
         raise UnknownDevice("Your device is unknown %s" % json.dumps(self._device))
 
-    def custom_initialize(self, extra_db):
+    async def custom_initialize(self, extra_db):
```

Comment:
..and this is probably a breaking change 🤔 Is that ok? Or is there a better way of handling it? I'll momentarily open a PR in https://github.com/bosch-thermostat/home-assistant-bosch-custom-component with the required changes.

Reply:
https://github.com/bosch-thermostat/home-assistant-bosch-custom-component/pull/425

Reply:
This would be a breaking change, as it needs changes to the component (as mentioned).
I see two options:

1. make `async` (as you have done)
2. keep sync create a second `async_custom_initialize()` for use in the HA component (if we want to be compatible with other users)

Personally, I'd go with 1) as the standard `initialize()` is async anyways.

Reply:
Thoughts about this @pszafer?